community-version.py

```diff
@@ -61,6 +75,7 @@ def handle_image_conversion(image_filepath) -> None:
         return
     image_ascii: str = convert_image_to_ascii(image)
     print(image_ascii)
+    return image_ascii
```

Comment:
Nice catch!

Now that you are returning a string would you be able to update the return type for `handle_image_conversion` to `str` like this?

```py
def handle_image_conversion(image_filepath) -> str:
  # ...
```

Everything else looks good to me.

Reply:
Good call. I added the return type to the handle_image_conversion function.

Reply:
Thanks! I will merge this now.